web/src/api/api.ts

```diff
@@ -170,6 +179,11 @@ function parseMetadata(data: d3.DSVRowArray): Metadata {
   );
 }
 
+export async function fetchBlocks(diffId: number, kmers: ObjMap<Kmer>): Promise<Array<Block>> {
```

Comment:
We'll probably want to keep `Diff.blocks`, but initialize it with `null`. This function could then populate the blocks in the given `Diff`.  This function could then be named `populateBlocks`.